express/code/blocks/pricing-table/pricing-table.js

```diff
@@ -199,6 +216,8 @@ const getId = (function idSetups() {
 }());
 
 export default async function init(el) {
+  headingCols = undefined;
```

Comment:
when declared but not initialized, javascript variables should be "undefined" already